far_mar.rb

```diff
@@ -0,0 +1,13 @@
+# gems your project needs
+require 'csv'
+
+# our namespace module
+module FarMar; end
+
+### rake won't work when I try to include these :(
```

Comment:
I think all you're missing is a `./` in your paths to make this work, i.e. `require './lib/farmar_market'`. It'd be worth figuring out as requiring the files here puts the module in charge of assembling what classes it needs. It'll also clean up your specs as you'll only need to require this file in `spec_helper`, and then require `spec_helper` in your specs, rather than requiring all the files all the time.